Va416x0/Os/SeggerTerminal/terminal.py

```diff
@@ -158,7 +159,11 @@ def __init__(self, speed=None):
 
     def _sync_connect_to_daemon(self):
         # Establish TCP connection to J-Link daemon via 127.0.0.1:19020
-        self.jlink.Open(HostIF=jlinksdk.HOST_IF.TCPIP, sIP="127.0.0.1")
+        try:
+            self.jlink.Open(HostIF=jlinksdk.HOST_IF.TCPIP, sIP="127.0.0.1")
+        except JLinkError as e:
+            print("Can not connect to JLink, attempt retry")
```

Comment:
Should use `file=sys.stderr` so it doesn't get confused with the actual `fprime-segger-rtt` output.